scripts/src/pushowners/owners.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import re
+from github import Github
+import yaml
+import sys
+import analytics
+sys.path.append('../')
+
+def getFileContent(files):
+    users_included="No"
+    provider_delivery="No"
+    vendor_name=""
+    chart_name=""
+    vendor_type=""
+    for changed_file in files:
+        # Load the YAML file
+        with open(changed_file) as file:
```

Comment:
We should require the file name to be charts/partners/<vendor>/OWNERS - we are not really interested in redhat or community charts because OWNERs file for these are not done by the redhat-mercury-bot. Also code would try to get data from the file even if it was not an OWNERS file.

Also the loop allows for multiple files but the function only returns data for one - the first found. I would suggest either we allow a PR to contain multiple OWNERS files and we do a metric for each or only get a metric if there is only one file in the push request and it is an  OWNERS file. Currently I would only expect the bot to include one OWNERS file so that would be my choice.

Reply:
I was actually having a loop as the modified and added files returned by jitterbit/get-changed-files@v1 is a list. 
But yeah as we are aware that the bot would push just one file , i have made changes to the code to accept a single file.